test_echo_server.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+
+import echo_server
+import socket
+
+def test_
+
```

Comment:
You are going to have a very hard time testing the relationship you built above in the two pair scripts.  Neither is really accessible since the sockets are peer-to-peer and you never return any values from either of the functions.